aiidalab_widgets_base/viewers.py

```diff
@@ -1049,7 +1049,7 @@ def apply_displayed_selection(self, _=None):
         else:
             self.wrong_syntax.layout.visibility = "visible"
 
-    def download(self, change=None):  # pylint: disable=unused-argument
+    def download(self, _change=None):
```

Comment:
I remember in other place this is just put as `_`.

Reply:
Done.